cmd/manager_logging.go

```diff
@@ -174,6 +174,14 @@ var (
 						Action: runAddSMTPLogger,
 					},
 				},
+			}, {
+				Name:   "log-sql",
+				Usage:  "Set LogSQL",
+				Action: runSetLogSQL,
+			}, {
+				Name:   "no-log-sql",
+				Usage:  "Switch off LogSQL",
+				Action: runUnSetLogSQL,
```

Comment:
IMO it seems that `gitea logging log-sql --off` more simple, and there will be only one `runSetLogSQLFlag` (instead of 3 functions).